nuxt.config.js

```diff
@@ -326,16 +332,51 @@ module.exports = {
   eslint: { cache: './node_modules/.cache/eslint' },
 };
 
+function proxyEmberOpts(target) {
+  return {
+    target,
+    secure:      !dev,
+    onProxyReq:  onProxyEmberReq,
+    onProxyReqWs,
+    onError,
+    onProxyRes,
+    // pathRewrite(path, req) {
+    //   console.log(`rewriting path: ${ path }`);
+
+    //   return path.replace('/ember/', '/');
+    // }
+  };
+}
+
 function proxyOpts(target) {
   return {
     target,
     secure: !dev,
     onProxyReq,
     onProxyReqWs,
-    onError
+    onError,
+    onProxyRes,
   };
 }
 
+function onProxyEmberReq(proxyReq, req) {
+  // console.log('ember request');
+  // console.log(proxyReq.path);
+
+  // if (proxyReq.path.startsWith('/ember')) {
+  //   proxyReq.path = proxyReq.path.substring(6);
+  // }
+  // console.log(proxyReq.path);
+  proxyReq.setHeader('x-api-host', req.headers['host']);
+  proxyReq.setHeader('x-forwarded-proto', 'https');
+}
+
+function onProxyRes(proxyRes, req, res) {
+  if (dev) {
+    proxyRes.headers['X-Frame-Options'] = 'ALLOWALL';
```

Comment:
This is fine for dev where the origins are different, but will need backend change to send SAMEORIGIN instead of DENY; the JS server is not involved in a real compiled build.  I asked Darren to change the API server.

Reply:
Thanks